extension/source/QuillPage/Wallet/Wallets/WalletWrapper.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import Button from '../../../components/Button';
+import { WalletSummary } from './WalletSummary';
+
+export interface IWallet {
+  address: string;
+  name: string;
+  ether: number;
+  networks: number;
+  tokens: number;
+}
+
+const wallets: IWallet[] = [
```

Comment:
ideally this data will come from an external source. Hence, mocked a probable approximate object.